src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder result = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        StringBuilder result = new StringBuilder(HEADER + dateFrom + DELIMITER + dateTo);
```